src/components/BingoBoardCard.vue

```diff
@@ -38,19 +40,23 @@
 </template>
 
 <script setup>
-import { ref, computed } from "vue";
+import { ref, computed, watch } from "vue";
 
-const bingoNumbers = ref([]);
+const bingoBoardNumbers = ref([]);
 const bingo = ref(null);
-const show = ref(false);
+const showRandomNumber = ref(false);
+const showWinner = ref(false);
 let startBingo = ref(null);
-const isBingo = ref(null);
+const startBtn = ref(true);
+const includedColumnIndexes = ref([]);
 
 const timeMultiplier = 0.2;

```

Comment:
Constants shall be named with Constants.
```suggestion
const TIME_MULTIPLIER = 0.2;

```